components/Icons/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+interface IconProps {
+  name: string;
```

Comment:
L5 - @andrewebdev could you change `name` to `icon` so that we don't have a type def error on L8?

Reply:
Yup, sorry I missed that one. For some reason the linter isn't working for me. I'll sort it when I have a moment.